composer.json

```diff
@@ -9,7 +9,7 @@
     }],
     "require": {
         "silverstripe/framework": "^4.0@dev",
-        "colymba/gridfield-bulk-editing-tools": "~2.1"
+        "colymba/gridfield-bulk-editing-tools": "^3.0@dev"
```

Comment:
I've had a beer. I could be wrong. I'll push a comnit without the specific and see what happens.